pkg/kubectl/cmd/get.go

```diff
@@ -315,5 +339,5 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 			continue
 		}
 	}
-	return utilerrors.Flatten(utilerrors.NewAggregate(allErrs))
+	return utilerrors.NewAggregate(allErrs)
```

Comment:
@smarterclayton 
> Printing out allErrs as a series of error lines would be consistent with what we do with Create.

By returning an `Aggregate` of the errors to `cmdUtil.CheckErr` after `RunGet` is done, `cmdUtil.CheckErr` ends up printing the series of errors in separate lines  per [MultipleErrors()](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/util/helpers.go#L139-L145)

Am I missing something?